TwitchDownloaderWPF/WindowQueueOptions.xaml.cs

```diff
@@ -336,31 +336,35 @@ private async void btnQueue_Click(object sender, RoutedEventArgs e)
                 {
                     PageChatRender renderPage = (PageChatRender)parentPage;
                     string folderPath = textFolder.Text;
-                    if (!String.IsNullOrWhiteSpace(folderPath) && Directory.Exists(folderPath))
+                    foreach (string fileName in renderPage.fileNames)
                     {
-                        ChatRenderTask renderTask = new ChatRenderTask();
-                        string fileFormat = renderPage.comboFormat.SelectedItem.ToString();
-                        string filePath = Path.Combine(folderPath, Path.GetFileNameWithoutExtension(renderPage.textJson.Text) + "." + fileFormat.ToLower());
-                        ChatRenderOptions renderOptions = MainWindow.pageChatRender.GetOptions(filePath);
-                        renderTask.DownloadOptions = renderOptions;
-                        renderTask.Info.Title = Path.GetFileNameWithoutExtension(filePath);
-                        var (success, image) = await InfoHelper.TryGetThumb(InfoHelper.THUMBNAIL_MISSING_URL);
-                        if (success)
+                        if (!String.IsNullOrWhiteSpace(folderPath) && Directory.Exists(folderPath))
```

Comment:
We shouldn't have to check that the folder path is not null on every run

Reply:
This line wasn't a code change, just whitespace change, so that code is already in there.

Reply:
I see. I'll be removing the null checks anyways throughout the file because `Directory.Exists()` internally has multiple null checks.